### PR TITLE
Added Recognition Page Link

### LINKED ISSUE #1520: Add Recognition Program Link to Community section
#### Current State of the Meshery.io site
image
image

#### Desired State of the Meshery.io site
Add [Recognition Program](https://badges.layer5.io/) link to the header and footer of the site, under the Community section

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Raj100679**
**Description**
Added Recognition link to community section

This PR fixes #1520 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Raj100679** [comment]
@iArchitSharma @leecalcote , I haven't set the external : true in navigation.yml because it was not getting aligned properly, If there are any changes, you can let me know?

File: .jekyll-metadata
Comment (iArchitSharma) [inline .jekyll-metadata:1]: **Do not push this file!**

File: Gemfile.lock
Comment (iArchitSharma) [inline Gemfile.lock:1]: **Do not push this file too!**

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
Good work @Raj100679 

Can you please do these changes:
1. Can you please revert the changes made to configuration files, you don't have to push configuration files
2. In the header navigation please place the **Recognition** Link below _Mailing Lists_ and above _Discuss_

image

**Comment by Raj100679** [comment]
@iArchitSharm I will do it tomorrow , I have to create a ppt for tomorrow

**Comment by iArchitSharma** [comment]
@Raj100679 you there?

**Comment by Raj100679** [comment]
@iArchitSharma will raise a pr in some time

**Pull request closed without merging**